multigeiger/multigeiger.ino

```diff
@@ -115,12 +115,12 @@ void setup() {
 void loop() {
   unsigned long time_difference;
   unsigned int HV_pulse_count;
-  char *sw;
+  Switches switches;
```

Comment:
sw was a bit short anyway, when i searched for that i regularly found password also...